src/app/beer_garden/db/schemas/garden_schema.py

```diff
@@ -19,17 +19,13 @@ class GardenBaseSchema(Schema):
     def validate_all_keys(self, post_load_data, original_data, **kwargs):
         # do not allow extraneous keys when operating on a dictionary
         if isinstance(original_data, dict):
-            extra_args = original_data.keys() - post_load_data.keys()
-
+            extra_args = post_load_data.keys() - original_data.keys()
```

Comment:
The changes made here in this commit are not actually related to any work in adding the buttons. But because this line was backwards, this code was firing and it revealed that the error messages were uninterpretable.